workflow/Snakefile

```diff
@@ -20,198 +20,30 @@ SBS_WELLS = list(sbs_samples_df["well"].unique())
 SBS_TILES = list(sbs_samples_df["tile"].unique())
 SBS_CYCLES = list(sbs_samples_df["cycle"].unique())
 # TODO: unrestrict wells and tiles after testing
-SBS_WELLS = ["A1"]
-SBS_TILES = [1]
+# SBS_WELLS = ["A1"]
+# SBS_TILES = [1]
 
 phenotype_samples_df = pd.read_csv(PHENOTYPE_SAMPLES_FP, sep="\t")
 phenotype_samples_df = phenotype_samples_df.sort_values(by=["well", "tile"])
 PHENOTYPE_WELLS = list(phenotype_samples_df["well"].unique())
 PHENOTYPE_TILES = list(phenotype_samples_df["tile"].unique())
 
 
+include: "targets/preprocess.smk"
 include: "rules/preprocess.smk"
 
 
+ALL_TARGETS = PREPROCESS_TARGETS_ALL
+
 if "sbs_process" in config:
 
+    include: "targets/sbs_process.smk"
```

Comment:
It makes sense overall. Targets and rules are separated out. I think this looks cleaner.